test/db/analysis/vars

```diff
@@ -873,7 +873,7 @@ pdf
 EOF
 EXPECT=<<EOF
             ;-- fn1:
-/ dbg.fn1 (int a, char *g, double q, struct Some *gg, some_t **out);
+/ dbg.fn1(int a,char * g,double q,struct Some * gg,some_t ** out);
```

Comment:
this is kinda weird.

Reply:
I think less spaces is fine, it allows to save the space (pun intended).  See also https://github.com/rizinorg/rizin/pull/3463#discussion_r1212608186